DBQnA/tests/test_compose_on_xeon.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+tgi_port=8008
+tgi_volume=$WORKPATH/data
+
+export model="meta-llama/Meta-Llama-3-8B-Instruct"
+export HUGGINGFACEHUB_API_TOKEN=${HUGGINGFACEHUB_API_TOKEN}
+export POSTGRES_USER=postgres
+export POSTGRES_PASSWORD=testpwd
+export POSTGRES_DB=chinook
+export TEXTTOSQL_PORT=9090
+export TGI_LLM_ENDPOINT="http://${ip_address}:${tgi_port}"
+
+
+function build_docker_images() {
+    echo $WORKPATH
+    OPEAPATH=$(realpath "$WORKPATH/../..")
+
+    echo "Building Text to Sql service..."
+    cd $OPEAPATH
+    rm -rf GenAIComps/
+    git clone https://github.com/opea-project/GenAIComps.git
+    cd $OPEAPATH/GenAIComps
+    docker build --no-cache -t opea/texttosql:latest -f comps/texttosql/langchain/Dockerfile .
+
+    echo "Building React UI service..."
+    cd $OPEAPATH/GenAIExamples/DBQnA/ui
+    docker build --no-cache -t opea/dbqna-react-ui:latest -f docker/Dockerfile.react .
+
+}
+
+function start_service() {
+
+    docker run --name test-texttosql-postgres --ipc=host -e POSTGRES_USER=${POSTGRES_USER} -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_DB=${POSTGRES_DB} -e POSTGRES_PASSWORD=${POSTGRES_PASSWORD} -p 5442:5432 -d -v $WORKPATH/docker_compose/intel/cpu/xeon/chinook.sql:/docker-entrypoint-initdb.d/chinook.sql postgres:latest
+
+    docker run -d --name="test-texttosql-tgi-endpoint" --ipc=host -p $tgi_port:80 -v ./data:/data --shm-size 1g -e HUGGINGFACEHUB_API_TOKEN=${HUGGINGFACEHUB_API_TOKEN} -e HF_TOKEN=${HUGGINGFACEHUB_API_TOKEN} -e model=${model} ghcr.io/huggingface/text-generation-inference:2.1.0 --model-id $model
+
+
+    unset http_proxy
```

Comment:
here unset http_proxy  was used so that docker does not refer to globally set proxy, instead  use the proxy setting that is set as environment variable. This is there is comps test as well. But it works fine without unset http_proxy, hence removed